src/cascadia/TerminalSettingsModel/TerminalSettingsSerializationHelpers.h

```diff
@@ -240,7 +240,7 @@ JSON_ENUM_MAPPER(::winrt::Windows::UI::Xaml::ElementTheme)
 JSON_ENUM_MAPPER(::winrt::Microsoft::Terminal::Settings::Model::NewTabPosition)
 {
     JSON_MAPPINGS(2) = {
-        pair_type{ "atTheEnd", ValueType::AtTheEnd },
+        pair_type{ "afterLastTab", ValueType::AfterLastTab },
         pair_type{ "afterCurrentTab", ValueType::AfterCurrentTab },

```

Comment:
We should keep an `atTheEnd` entry here to ensure that existing users' settings still work:
```cpp
pair_type{ "atTheEnd", ValueType::AfterLastTab },
```

We can ensure that `afterLastTab` is preferred over `atTheEnd` by putting that legacy mapping after the new one (for instance at the end of the list).

Reply:
Is that what "No need for legacy converter" in the issue was about?

Reply:
Oh, yeah, we don't need to include the legacy value at all -- this feature didn't ship outside of limited selfhost!